script.js

```diff
@@ -1,28 +1,86 @@
-var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "Jack", "Queen", "King", "Ace"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
   deck: [],
   players: [],
   hands: [],
   buildDeck: function(){
-
+      for (var myRank = 0; myRank < values.length; myRank++) {
+        for (var mySuit = 0; mySuit < suits.length; mySuit++) {
+          this.deck.push({ "rank" : values[myRank], "suit" : suits[mySuit]});
```

Comment:
This is great, but might also be a useful time to grab the `values`'s index  for each `card` so as to determine a numerical `weight` property